viewstate/viewstate.py

```diff
@@ -1,12 +1,17 @@
 from base64 import b64decode, b64encode
 from binascii import Error as BinAsciiError
+try:
+    from Crypto.Cipher import AES, DES3, DES
+except ImportError:
+    from Cryptodome.Cipher import AES, DES3, DES
```

Comment:
@rpimonitrbtch can you please describe where these dependencies come from? Are they expected to be built-in to the users Python installation? Do we need to bring them in as dependencies in `setup.py`?